docs/Pipeline-Documentation.md

```diff
@@ -52,12 +60,36 @@ An assortment of other types of utility pipelines.  Utility piplines are named `
 
 ## Training
 
-Run model training on ground truth annotations you've created in order to generate a reusable pipe.
+Run model training on ground truth annotations.  Currently, training configurations are available to do object detection, object classification, and full-frame classification.  Tracker training will be added in a future update.
+
+### Important notes
+
+* Object detector training configurations require annotations labeled `fish`.
+* Full-frame classifiers can be trained on arbitrary multi-class labels.  It's helpful to start with `empty frame lbls` utility pipe and add type annotations to each generated frame.
+* Object classifiers are trained on bounding boxes with arbitrary multi-class labels.
+
+### Overview
+
+* SVM ([Support Vector Machine](https://en.wikipedia.org/wiki/Support-vector_machine)) configurations are usable with the smallest amount of ground-truth and train relatively quickly.
+* [NetHarn](https://gitlab.kitware.com/computer-vision/netharn) is a pytorch deep learning framework that requires more input data: on the order of thousands of target examples.  There are two architectures used.  Netharn models can take up to several days to train.
+    * Cascade Faster R-CNN (cfrnn)
+    * ResNet (Residual Network)
```

Comment:
Also mask Rcnn might want to say mask rcnn useful for pixel class + detection (boxes), CFRNN just detection (boxes), resnet full frame or secondary object classification